Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/ListBetaTestersCommand.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBetaTestersCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "list",
+        abstract: "List beta testers")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Option(default: "",
```

Comment:
if it's optional, for `case let(bundleId, _, _) where !bundleId.isEmpty:`  I have to change to 
```
case let (bundleId, _, _) where bundleId != nil:
    self = .listByApp(bundleId: bundleId!)
```
because I don't know how to use `if let` or `guard` to removing the force unwrapping`!`?